scripts/restore

```diff
@@ -13,12 +13,12 @@ set -eu
 # IMPORT GENERIC HELPERS
 #=================================================
 
-# if [ ! -e _common.sh ]; then
-# 	# Rapatrie le fichier de fonctions si il n'est pas dans le dossier courant
-# 	cp ../settings/scripts/_common.sh ./_common.sh
-# 	chmod a+rx _common.sh
-# fi
-# source _common.sh
+if [ ! -e _common.sh ]; then
+	# Rapatrie le fichier de fonctions si il n'est pas dans le dossier courant
+	cp ../settings/scripts/_common.sh ./_common.sh
+	chmod a+rx _common.sh
+fi
```

Comment:
Why do you can't do something like that : `source ../settings/scripts/_common.sh`

I did this in all of my app without problem.

Reply:
It's an old way to to that, it was probably because we needed a chmod +x

Reply:
So, could we remove it to make the code more clean ?

Reply:
If it works for you, yes we can do that